### PR TITLE
Duplicate columns - bug fix

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgDuplicateColumns.vb [hunk 1]
```diff
@@ -94,6 +96,10 @@ Public Class dlgDuplicateColumns
         End If
     End Sub
 
+    Private Sub ReopenDialog()
+        ucrInputColumnName.bAllowNonConditionValues = True
+    End Sub
```
Comment (lilyclements) [inline instat/dlgDuplicateColumns.vb]: Adding on ReopenDialog() to prevent a developer error occurring when reopening this dialog

Comment (dannyparsons) [inline instat/dlgDuplicateColumns.vb]: Should this not be done on initialise then if it's needed?

Comment (lilyclements) [inline instat/dlgDuplicateColumns.vb]: If I set on initialise I still get the error when reopening?

Comment (dannyparsons) [inline instat/dlgDuplicateColumns.vb]: Ok with this for now then but should come back to investigate.

Comment (dannyparsons) [inline instat/dlgDuplicateColumns.vb]: Can you add a comment that this a temp fix?

Comment (lilyclements) [inline instat/dlgDuplicateColumns.vb]: Sure, added the comment :)

**Pull request merged**